hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -209,7 +208,7 @@ public static String randomNameOwnedBy(HazelcastInstance instance, String prefix
         for (; ; ) {
             String id = prefix + randomString();
             Partition partition = partitionService.getPartition(id);
-            if (comparePartitionOwnership(false, localMember, partition)) {
+            if (comparePartitionOwnership(true, localMember, partition)) {
```

Comment:
why not have something like `boolean isOwnedBy(Partition p, Member m);` instead of the `comparePartitionOwnership` ? 

I found the `comparePartitionOwnership()` a bit confusing -> that's likely a root cause of the wrong boolean flag.